cmd/kube-apiserver/app/server.go

```diff
@@ -568,7 +570,9 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 	}
 
 	if s.Etcd.EncryptionProviderConfigFilepath != "" {
-		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(s.Etcd.EncryptionProviderConfigFilepath)
+		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(
```

Comment:
Would recommend creating a new sub package of `cmd/kube-apiserver/app` called `encryption`, define a function in there that handles GetTransformerOverrides() and accepts as input `CloudProvider`.